src/main/java/com/systemmeltdown/robotlib/subsystems/SingleSpeedTalonDriveSub.java

```diff
@@ -0,0 +1,194 @@
+package com.systemmeltdown.robotlib.subsystems;
+
+import java.util.ArrayList;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatusFrame;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.systemmeltdown.robotlib.util.PIDValues;
+import com.systemmeltdown.robotlib.util.Utility;
+
+public class SingleSpeedTalonDriveSub extends SkidSteerDriveSubBase {
+    WPI_TalonSRX m_rightMaster;
+    WPI_TalonSRX m_leftMaster;
+    WPI_TalonSRX[] m_rightSlaves;
+    WPI_TalonSRX[] m_leftSlaves;
+    int m_timeout;
+    int m_talonPidPrimary;
+    double m_driveMotorDeadband;
+    double m_driveRampSeconds;
+
+    int m_talonSlotVelocity;
```

Comment:
These slots were very specific to the setup for last year's robot. We need to put some thought into how we want to allocate the Talon slots for this.